src/workerd/server/workerd-api.c++

```diff
@@ -583,7 +583,8 @@ static v8::Local<v8::Value> createBindingValue(
     }
 
     KJ_CASE_ONEOF(ns, Global::AnalyticsEngine) {
```

Comment:
Nit (here and elsewhere); I think `ns` above stands for "namespace", so it is a little bit confusing here. How about `ae`?